packages/synapse-react-client/stories/TwoFactorAuthSettingsPanel.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Meta, StoryObj } from '@storybook/react'
+import { displayToast } from '../src/lib/containers/ToastMessage'
+import TwoFactorAuthSettingsPanel from '../src/lib/containers/auth/TwoFactorAuthSettingsPanel'
+import { Paper } from '@mui/material'
+
+const meta = {
+  title: 'Authentication/TwoFactorSettingsPanel',
+  component: TwoFactorAuthSettingsPanel,
+  render: args => {
+    return (
+      <Paper sx={{ p: 7, mx: 'auto', width: '720px' }}>
```

Comment:
Wrapped in paper for Storybook, which we would probably do in SageAccountWeb, while we can wrap it in the lightbox in SWC